CHANGELOG.md

```diff
@@ -22,11 +22,8 @@
 
 🔧 Fixes:
 
-- Pull Request Title goes here
-
-  Description goes here (optional)
-
-  ([PR #N](https://github.com/alphagov/govuk-frontend/pull/N))
+- Replace global `js-hidden` class with tabs panel modifier
```

Comment:
Just to be super clear, what about something like:

> Replace conflicting `js-hidden` class used within the tabs component with a new modifier class. Because this class is defined and used within the JavaScript, no markup changes are required.

Thoughts?